src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -232,7 +233,7 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
                 do_CAR=do_CAR,
                 invert_sign=invert_sign,
                 device=device,
-                save_preprocesed_copy=save_preprocessed_copy,  # this kwarg is correct (typo)
+                save_preprocessed_copy=save_preprocessed_copy,  # this kwarg is correct (typo)
```

Comment:
@mhhennig this was a typo in a previous KS version (see https://github.com/SpikeInterface/spikeinterface/pull/3316). Can you revert the KS changes? It's better to have a dedicated PR to handle them

Reply:
Ok